src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -127,9 +127,14 @@ class ObjectField extends React.Component {
         );
         // Use div or dl to wrap content for array type schemas (e.g. bank info)
         // fixes axe issue on review-and-submit
-        divWrapper = objectFields.some(
-          name => uiSchema?.[name]?.['ui:options']?.volatileData,
-        );
+        divWrapper = objectFields.some(name => {
+          const options = uiSchema?.[name]?.['ui:options'] || {};
+          return (
+            options.volatileData || // ReviewCardField
+            options.customTitle || // SelectArrayItemsWidget
+            options.addAnotherLabel // fileUiSchema
+          );
+        });
```

Comment:
I'd love to get this documented. Unfortunately, we don't really have a good place for this. :disappointed: 

cc @Clark-W-Griswold something to keep in mind when we go to document all the things. This kind of tight coupling between the `ObjectField` and custom widgets is no good. I'm not familiar enough with the problem this is solving anymore (though I'm pretty sure I was the one to write the first iteration of this :grimacing: ), so I don't have a better solution yet.

Reply:
I thought about adding a new `ui:option` specifically meant to make it wrap the content in a `div` vs a `dl`, but then I'd end up modifying a lot of forms. Maybe this is a better solution?

Reply:
No, this is fine for now. We'll want to figure out a more generic way to determine what element should be rendered. It'll likely mean defining bona-fide interfaces for custom widgets, but we aren't in a place to make that a reality yet.